packages/toolkit/src/autoBatchEnhancer.ts

```diff
@@ -62,6 +55,13 @@ export const autoBatchEnhancer =
 
     const listeners = new Set<() => void>()
 
+    // requestAnimationFrame won't exist in SSR environments.
+    // Fall back to a vague approximation just to keep from erroring.
+    const rAF =
+      typeof window !== 'undefined' && window.requestAnimationFrame
+        ? window.requestAnimationFrame
+        : createQueueWithTimer(10)
+
```

Comment:
At this point, this could probably be inlined in line 69.

Reply:
Makes sense, I have pushed this change.